content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -397,4 +397,8 @@ The design documents for
 [node affinity](https://git.k8s.io/community/contributors/design-proposals/scheduling/nodeaffinity.md)
 and for [inter-pod affinity/anti-affinity](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md) contain extra background information about these features.
 
+Once a Pod is assigned to a Node, the kubelet runs the Pod and allocates node-local resources.
+The [topology manager](/docs/tasks/administer-cluster/topology-manager/) takes part in node-level
```

Comment:
Just a remark.
If this gets merged as-is, I would like to log a low-priority issue to document Topology Manager in the Concepts section as well. Any time that the docs reference a concept there should be a page in Concepts that can be linked to.

(@fejta-bot might eventually closed it unfixed, which isn't the end of the world if it happens).

None of the above means this isn't good to merge.

Reply:
@sftim So, a topology manager specific page in the concepts folder? Is there an example one I could look at or you could point me towards?

Reply:
Maybe, for now: hyperlink, to `/docs/concepts/architecture/nodes/#node-topology`

@lmdaly Do you think that's better or worse?

Reply:
Added, please take a look!